schemas/user_tasks.proto

```diff
@@ -47,43 +50,21 @@ message UserTaskRun {
   // field, or a `oneof user_task_source` field. However, note that such
   // a change would be fine from the API Compatibility perspective.
   NodeRunId node_run_id = 11;
-
-  oneof owner {
-    UserGroup user_group = 12;
-    User user = 13;
-  }
 }
 
 message AssignUserTaskRunRequest {
   UserTaskRunId user_task_run_id = 1;
 
   bool override_claim = 2;
 
-  oneof assignee {
-    User user = 3;
-    UserGroup user_group = 4;
-  }
-}
-
-message UserTaskFieldResult {
-  string name = 1;
-  optional VariableValue value = 2;
-}
-
-message UserTaskResult {
-  repeated UserTaskFieldResult fields = 1;
-}
-
-message SaveUserTaskRun {
-  UserTaskResult result = 1;
-  string user_id = 2;
-  repeated UserTaskFieldResult results = 3;
+  optional string user_group = 3;
+  optional string user_id = 4;
 }
 
 message CompleteUserTaskRunRequest {
   UserTaskRunId user_task_run_id = 1;
 
-  UserTaskResult result = 2;
+  map<string, VariableValue> results = 2;
   string user_id = 3;
```

Comment:
This is interesting, in the old version we didn't pass in user group id...I think that's where the confusion came with disappearing groups.